Doc/whatsnew/3.12.rst

```diff
@@ -124,3 +119,25 @@ Changes in the Python API
   The group name in bytes patterns and replacement strings can now only
   contain ASCII letters and digits and underscore.
   (Contributed by Serhiy Storchaka in :gh:`91760`.)
+
+
+Build Changes
+=============
+
+
+C API Changes
+=============
+
+New Features
+------------
+
+Porting to Python 3.12
+----------------------
+
+Deprecated
+----------
+
+Removed
+-------
+
+* :c:func:`PyFrame_BlockSetup` and :c:func:`PyFrame_BlockPop` have been removed.
```

Comment:
It's probably enough to just list the names, since they are now under the "Removed" section.
```suggestion
* :c:func:`PyFrame_BlockSetup` and :c:func:`PyFrame_BlockPop`.
```

Reply:
I prefer to leave it as it is. See What's New in Python 3.11: https://docs.python.org/dev/whatsnew/3.11.html#id8

Also, this PR is only about moving items. Not rephrasing them.

Reply:
@ezio-melotti: Feel free to propose a follow-up PR if you want to rephase it ;-) Personally, I like to write sentences which can be read aloud.